xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-macro/src/main/resources/ApplicationResources.properties

```diff
@@ -77,6 +77,8 @@ livedata.selection.infoBar.selectedCount={0} entries selected
 livedata.selection.infoBar.allSelected=All entries selected
 livedata.selection.infoBar.allSelectedBut=All entries selected but {0}
 
+livedata.pagination.label={0} live table page info
+livedata.pagination.label.empty=Live table page info
```

Comment:
Why does this talk about "Live table" and not "Live data"? Is that consistent with the rest of the UI?

Reply:
That's a bad mistake, fixing it, thanks.